### DISCUSSION TITLE
Hello everybody my name is markiplier and I am officially back

### DISCUSSION THREAD

**Discussion by moxie-coder** [FunkinCrew/Funkin#2116] [General]
yes
yes!
YEEE--

**Comment by fnfporterhi** [comment]
HELLO everybody my name is markiplier…. WAS THAT THE BITE OF 8-

**Reply by moxie-coder** [reply]
lmao

**Comment by moxie-coder** [comment]
![mark-markiplier](https://github.com/FunkinCrew/Funkin/assets/124418090/4ac67c88-20c4-4e2a-898f-cb8dfb71a422)

**Comment by fnfporterhi** [comment]
what

**Reply by moxie-coder** [reply]
markiplier with a gun
